airbyte-ci/connectors/live-tests/src/live_tests/commons/proxy.py

```diff
@@ -112,8 +112,12 @@ async def bind_container(self, container: dagger.Container) -> dagger.Container:
             dagger.Container: The container with the proxy service bound and environment variables set.
         """
         cert_path_in_volume = "/mitmproxy_dir/mitmproxy-ca.pem"
-        requests_cert_path = "/usr/local/lib/python3.9/site-packages/certifi/cacert.pem"
         ca_certificate_path = "/usr/local/share/ca-certificates/mitmproxy.crt"
+
+        python_version_output = (await container.with_exec(["python", "--version"], skip_entrypoint=True).stdout()).strip()
+        python_version = python_version_output.split(" ")[-1]
+        python_version_minor_only = ".".join(python_version.split(".")[:-1])
+        requests_cert_path = f"/usr/local/lib/python{python_version_minor_only}/site-packages/certifi/cacert.pem"
```

Comment:
Should we add a check here that verifies that this file exists, and fails loudly if it does not? That way we could have an error message on telemetry that certificate failed?